app/controllers/devise_guest_controllers_helpers_decorator.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
```

Comment:
An override comment could be useful.

Reply:
I swear there was one. but yes I 100% agree with you